tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/wifi.py

```diff
@@ -5,10 +5,12 @@
 
 try:
     from marionette import Wait
+    from marionette import expected
```

Comment:
Please update this to fallow the new import pattern.

https://github.com/mozilla-b2g/gaia/blob/master/tests/python/gaia-ui-tests/gaiatest/apps/phone/app.py#L6